filebeat/module/mysql/slowlog/_meta/fields.yml

```diff
@@ -31,6 +35,55 @@
       type: long
       description: >
         The connection ID for the query.
+    - name: thread_id
+      type: long
+      description: >
+        ID of the thread executing this query.
+    - name: schema
+      type: keyword
+      description: >
+        The schema where the slow query was executed.
+    - name: current_user
+      type: keyword
+      description: >
+        Current authenticated user, used to determine access privileges. Can differ from the value for user.
+
+    - name: query_cache_hit
+      type: keyword
+      description: >
+        Whether the query cache was hit.
+    - name: tmp_table
+      type: keyword
+      description: >
+        Whether a temporary table was used to resolve the query.
+    - name: tmp_table_on_disk
+      type: keyword
+      description: >
+        Whether the query needed temporary tables on disk.
+    - name: filesort
+      type: keyword
+      description: >
+        Whether filesort optimization was used.
+    - name: filesort_on_disk
+      type: keyword
+      description: >
+        Whether filesort optimization was used and it needed temporary tables on disk.
+    - name: priority_queue
+      type: keyword
+      description: >
+        Whether a priority queue was used for filesort.
+    - name: full_scan
+      type: keyword
+      description: >
+        Whether a full table scan was needed for the slow query.
+    - name: full_join
+      type: keyword
+      description: >
+        Whether a full join was needed for the slow query (no indexes were used for joins).
```

Comment:
Actually most of the fields added could be boolean. Any proposal to change all `Yes`/`No` values for `true`/`false`?

Reply:
https://github.com/elastic/beats/pull/9731/commits/de643bc1b43b604f6fd7e698cb25ec264b6c300a

Reply:
I like the change to bool. Not sure if there is a nicer solution. Perhaps a dedicated processor on the ES side for such things? :-D

Reply:
A replacer could be a good processor :slightly_smiling_face:
And it could be also a good addition if processors accepted lists of fields, this could have been also done with a couple of `gsub` processors if it accepted lists of processors.

But well, at the end this script doesn't look so bad :smile: